src/app/zap-templates/zcl/data-model/chip/access-control-cluster.xml

```diff
@@ -49,13 +51,15 @@ limitations under the License.
   </enum>
 
   <struct name="AccessControlTargetStruct">
+    <cluster code="0x0752"/> <!-- Joint Fabric Datastore -->
```

Comment:
Same deal.  Spec needs to be fixed, and once it's fixed we can figure out what the XML here should look like.

Reply:
Copied the enum definition to the JF datastore cluster and submitted the update in the most recent change set.